egret/model_library/transmission/bus.py

```diff
@@ -150,10 +146,11 @@ def declare_eq_p_balance_ed(model, index_set, bus_p_loads, gens_by_bus, bus_gs_f
     if rhs_kwargs:
         for idx,val in rhs_kwargs.items():
             if idx == 'include_feasibility_slack':
-                p_expr += eval("m." + val)[_system_idx]
+                p_expr -= eval("m." + val)
```

Comment:
It looks like the power balance slack variable defined here is a singleton, e.g., one real that can be negative or positive. Assuming that is correct, it’s probably worth looking at splitting it into positive and negative components, such that it can be added to relevant optimization objectives if desired.

Reply:
Ah, well I think I was trying to let it be sign indefinite, so that one could define slack == slack^+ - slack^- where slack^+, slack^- >= 0. However, I've updated the code to make this relationship explicit in the balance (and so one does not have to define an additional constraint).

Reply:
I'm happy, especially given the direction @bknueven is moving on the UC front. I mainly brought it up to make sure we think about unification across the different model classes, and that is already happening.